src/functions.ts

```diff
@@ -64,6 +64,8 @@ const translateType = (
             return prop.$.type;
         case "widgets":
             return "ReactNode";
+        case "file":
+            return preview && !isChild ? "FileValue" : "DynamicValue<FileValue>";
```

Comment:
We are not testing the isChild part anywhere where the outcome would be "DynamicValue<FileValue>"

Reply:
Just pushed new tests using a list of files that cover this case